src/design-guidelines/action-buttons/examples/labels.tsx

```
@@ -8,39 +8,33 @@ import { Component, h } from '@stencil/core';
-                        heading="Don't"
-                    ></limel-header>
-                    <div class="fake-dialog-container shows-full-dialog">
-                        <div class="fake-dialog">
-                            <p>
-                                <b>Bad</b> usage of labels
-                            </p>
-                            <div class="action-bar">
-                                <limel-button label="Close" />
-                                <limel-button label="Save" primary={true} />
-                            </div>
+            <limel-example-do-do-not>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)